app/services/gitter.rb

```diff
@@ -0,0 +1,33 @@
+require 'json'
+
+module Gitter
+  class Client
+    include HTTParty
```

Comment:
Can we use Faraday instead please? 

Faraday already is a dependency of a few of our other gems so it makes sense to use it over adding another similar gem.